dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionGroupRestControllerIT.java

```diff
@@ -0,0 +1,2094 @@
+        Group submittersGroup = collectionService.createSubmitters(context, collection);
+        authorizeService.addPolicy(context, parentCommunity, Constants.ADMIN, eperson);
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isNoContent());
+
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isNoContent());
+    }
+
+    // This is currently not supported in DSpace API
+    @Ignore
+    @Test
+    public void deleteCollectionSubmittersGroupTestCollectionAdmin() throws Exception {
+        Group submittersGroup = collectionService.createSubmitters(context, collection);
+        authorizeService.addPolicy(context, collection, Constants.ADMIN, eperson);
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isNoContent());
+
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isNoContent());
+    }
+
+    @Test
+    public void deleteCollectionSubmittersGroupUnAuthorizedTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Group submittersGroup = collectionService.createSubmitters(context, collection);
+        context.restoreAuthSystemState();
+
+        getClient().perform(delete("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                   .andExpect(status().isUnauthorized());
+    }
+
+    @Test
+    public void deleteCollectionSubmittersGroupForbiddenTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Group submittersGroup = collectionService.createSubmitters(context, collection);
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/submittersGroup"))
+                        .andExpect(status().isForbidden());
+    }
+
+
+    @Test
+    public void deleteCollectionSubmittersGroupNotFoundTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Group submittersGroup = collectionService.createSubmitters(context, collection);
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(eperson.getEmail(), password);
+
+        getClient(token).perform(delete("/api/core/collections/" + UUID.randomUUID() + "/submittersGroup"))
+                        .andExpect(status().isNotFound());
+    }
+
+    private Group createDefaultReadGroup(String itemGroupString, int defaultItemRead)
```

Comment:
this seems something that need to be moved to the collectionservice

Reply:
This was moved, we also altered the code in the Repository to make use of this new service method.